gradle.properties

```diff
@@ -1 +1 @@
-version=0.1-SNAPSHOT
\ No newline at end of file
+version=0.1-SNAPSHOT
```

Comment:
Should this be reverted?

Reply:
I didn't make a change here (apart from trailing newline) - it was already 0.1-SNAPSHOT

Reply:
No problem, I got mixed up looking at the github UI